src/askui/tools/screen_switch_tool.py

```diff
@@ -0,0 +1,45 @@
+from askui.models.shared.tools import Tool
+from askui.tools.agent_os import AgentOs, DisplayInformation
+
+
+class ScreenSwitchTool(Tool):
+    """
+    Tool to change the screen.
+    """
+
+    def __init__(self, agent_os: AgentOs) -> None:
+        # We need to determine the number of displays available to provide context
+        # to the agent indicating that screen switching can only be done this number
+        # of times.
+        displays: list[DisplayInformation] = agent_os.get_display_information().displays
+
+        super().__init__(
+            name="screen_switch",
+            description=f"""
+            This tool is useful for switching between multiple displays to find
+            information not present on the current active screen.
+            If more than one display is available, this tool cycles through them.
+            Number of displays available: {len(displays)}.
+            """,
+        )
+        self._agent_os: AgentOs = agent_os
+        self._displays: list[DisplayInformation] = displays
```

Comment:
Decent idea of listing the displays here and including it in the tool description although that does not match what is specified in the issue. The problem I see with this is that it is too static as (the number of) displays may change throughout the execution. So I think going with an extra tool for listing displays the better approach. WDYT?